packages/capabilities/src/types.ts

```diff
@@ -468,6 +468,11 @@ export interface StoreAddSuccessUpload extends StoreAddSuccessResult {
   status: StoreAddSuccessStatusUpload
   url: ToString<URL>
   headers: Record<string, string>
+  /**
+   * http method to use in request to url.
+   * If undefined, try "PUT".
+   */
+  method?: string
```

Comment:
We should be explicit.

```suggestion
  /**
   * HTTP method to for the request to `url`.
   */
  method: 'PUT'
```

Reply:
I would like the type to allow the server to instruct the client to send any http method, not just 'PUT'. It seems like typing this as only 'PUT' would move away from that goal

Reply:
> We should be explicit.

what's bad about it being explicitly any syntactically valid http method (i.e. something that metches the type of `fetch` `options.method`) vs being explicitly PUT?

Reply:
Ah, I meant explicit like not optional :) 

I added PUT because that's all we use right now...but any HTTP method I guess is fine for the type...PUT|POST - are there others it could be?